bot/seasons/evergreen/8bitify.py

```diff
@@ -0,0 +1,54 @@
+from io import BytesIO
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+
+class EightBitify(commands.Cog):
+    """Make your avatar 8bit!"""
+
+    def __init__(self, bot):
```

Comment:
This method is missing function annotations, both for the `bot` parameter as well as the return value.

With your current imports, this could be:

```suggestion
    def __init__(self, bot: commands.Bot) -> None:
```